.changeset/tall-buckets-change.md

```diff
@@ -0,0 +1,284 @@
++ --salt-navigable-textDecoration-selected
+
+- --salt-text-primary-foreground
+- --salt-text-primary-foreground-disabled
+- --salt-text-secondary-foreground
+- --salt-text-secondary-foreground-disabled
++ --salt-content-primary-foreground
++ --salt-content-primary-foreground-disabled
++ --salt-content-secondary-foreground
++ --salt-content-secondary-foreground-disabled
+
+- --salt-text-background-selected
++ --salt-content-foreground-highlight
+
+- --salt-navigable-primary-background-hover
++ --salt-navigable-background-hover
+
+- --salt-accent-fontWeight
+- --salt-accent-fontSize
+- --salt-accent-lineHeight
++ --salt-text-notation-fontWeight
++ --salt-text-notation-fontSize
++ --salt-text-notation-lineHeight: 10px; /* Value previously 11px */
+
+- --salt-track-borderWidth
+- --salt-track-borderWidth-active
+- --salt-track-borderWidth-complete
+- --salt-track-borderWidth-incomplete
++ --salt-size-border-strong
+
+- --salt-track-borderColor
++ --salt-palette-neutral-secondary-border
+
+- --salt-selectable-primary-foreground-selected
++ --salt-actionable-primary-foreground-active
+
+- --salt-selectable-primary-foreground-selectedDisabled
++ --salt-color-white-fade-foreground
+
+- --salt-selectable-primary-background
+- --salt-selectable-primary-background-disabled
+- --salt-selectable-primary-background-hover
+- --salt-selectable-primary-background-selected
+- --salt-selectable-primary-background-selectedDisabled
++ --salt-palette-interact-background
++ --salt-palette-interact-background-disabled
++ --salt-palette-interact-primary-background-hover
++ --salt-palette-interact-primary-background-active
++ --salt-palette-interact-primary-background-activeDisabled
+
+- --salt-selectable-secondary-foreground-hover
+- --salt-selectable-secondary-foreground-selected
+- --salt-selectable-secondary-foreground-selectedDisabled
++ --salt-palette-interact-secondary-foreground-hover
++ --salt-palette-interact-secondary-foreground-active
++ --salt-palette-interact-secondary-foreground-activeDisabled
+
+- --salt-selectable-secondary-background
+- --salt-selectable-secondary-background-disabled
+- --salt-selectable-secondary-background-hover
+- --salt-selectable-secondary-background-selected
+- --salt-selectable-secondary-background-selectedDisabled
++ --salt-palette-interact-background
++ --salt-palette-interact-background-disabled
++ --salt-palette-interact-secondary-background-hover
++ --salt-palette-interact-secondary-background-active
++ --salt-palette-interact-secondary-background-activeDisabled
+```
+
+- Deprecated tokens with no direct replacement tokens, use values suggested in comments
+
+```diff
+- --salt-taggable-cursor-hover /* Use `pointer` */
+- --salt-taggable-cursor-active /* Use `pointer` */
+- --salt-taggable-cursor-disabled /* Use `not-allowed` */
+
+- --salt-taggable-background /* Use --salt-palette-interact-primary-background */
+- --salt-taggable-background-hover /* Use --salt-palette-interact-primary-background-hover */
```

Comment:
dont suggest to use palette token directly, it can change. better to say the exact rgb value